app/views/planning_applications/review/tasks/index.html.erb

```diff
@@ -36,20 +36,26 @@
 <% if @planning_application.application_type.consultation? %>
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-full">
-        <%= render "review_consultation" %>
+      <%= render "planning_applications/review/tasks/review_consultation" %>
```

Comment:
I wonder if we can make some of these `render` calls less longwinded by moving the partials into the right place for it to detect them without a full path?

Reply:
If we are building out separate controllers for these forms, I think when rendering the page again due to an error we'll need the full context of the partial.